### PR TITLE
Disable pre-fetching by default

### PR DESCRIPTION
**Pull request by MrBartusek**
dragon Pre-Fetching is really a wonderful feature of this library. It really speed up any operations on any ddragon data. The problem I have with the current setup is that it's enabled by default. When users run the example form getting started they need to wait a minute for it to run:

image

Obviously, the getting started guide explains that there is an initialization procedure that takes time and the client will appear stuck but, honestly, most people will just run the code and don't read the guide any further. This creates confusion since generally, API wrappers initialize themselves in just a couple of seconds. That's why I think it should be an opt-in feature.

### CODE REVIEW

**Review by TheDrone7 — APPROVED** [review]
This makes sense, though I do tend to prefer it being enabled by default, adding a shortcut will make this less painful.

**Comment by MrBartusek** [comment]
@TheDrone7 certainly #14 will make it a lot less painful but still, the end user just starting with library won't be fiddling with client settings even if they are just simple two faleses.

Also another question, I just wonder since I don't see many people doing it. Why are you closing PRs and committing them instead of just merging? I think just normal GitHub merges are first of all faster since you can just use 2 clicks to merge them. Second of all, I don't get any credit for the changes the way you are doing it😢

**Comment by TheDrone7** [comment]
I just wanted to see the difference locally because my editor was not really causing me any issues. And since I'd already done it, I just committed and closed PR.

BTW, can you merge this after my approval or would I have to merge it, never had any PRs on my repos by others before lmao

**Comment by TheDrone7** [comment]
I just wanted to see the difference locally because my editor was not really causing me any issues. And since I'd already done it, I just committed and closed PR.

BTW, can you merge this after my approval or would I have to merge it, never had any PRs on my repos by others before lmao

**Comment by MrBartusek** [comment]
> BTW, can you merge this after my approval or would I have to merge it, never had any PRs on my repos by others before lmao

I can't directly write any code to any branch in any way in this repo. All PRs must me merged by someone with write access. I can do whewer I want with https://github.com/MrBartusek/shieldbow but not here. I can really only open and close my Prs and issues and leave reviews on other PRs which are even marked that they are not coming from someone with write access. That's pretty much all.

**Pull request merged**
